src/main/java/duke/task/Deadlines.java

```diff
@@ -0,0 +1,33 @@
+package duke.task;
+
+/**
+ * sub-class of Task to represents a task with deadline.
+ */
+public class Deadlines extends Task {
```

Comment:
Perhaps this can be named Deadline instead of Deadlines, since it is a single Task